### PR TITLE
build: Bump version to 21.04.0

### PR DESCRIPTION
**Pull request by Emiller88**
Recent stable release https://github.com/nextflow-io/nextflow/releases/tag/v21.04.0

Mainly to test the CI against it, and to open the discussion for whether we need to support the older edge version.

### CODE REVIEW

**Review by drpatelh — APPROVED** [review]
Awesome! Thanks. You got all of them right? Feel free to merge ;)

**Comment by drpatelh** [comment]
Latest stable version is fine for now to restrict the number of tests we are doing. It is also the latest at the mo so fine I think.

**Comment by Emiller88** [comment]
The only issue I saw was the README still has some stuff on edge releases. Not sure whether to cut that out or leave it.

File: README.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ If the module doesn't exist on `nf-core/modules`:
 We have implemented a number of commands in the `nf-core/tools` package to make it incredibly easy for you to create and contribute your own modules to nf-core/modules.
 
 1. Install the latest version of [`nf-core/tools`](https://github.com/nf-core/tools#installation) (`>=1.13`)
-2. Install [`nextflow`](https://nf-co.re/usage/installation) (`>=20.11.0-edge`; see [Nextflow edge releases](#nextflow-edge-releases))
+2. Install [`nextflow`](https://nf-co.re/usage/installation) (`>=21.04.0`; see [Nextflow edge releases](#nextflow-edge-releases))
```
Comment (drpatelh) [inline README.md:170]:
```suggestion
2. Install [`Nextflow`](https://nf-co.re/usage/installation) (`>=21.04.0`; see [installation docs](https://www.nextflow.io/docs/latest/getstarted.html#installation))
```

Comment (drpatelh) [reply README.md:170]: We can remove all of the edge release docs here now and replace it with a link to main NF installation docs as I have done above. I kick-started them being added to the main NF docs so we can just link to those everywhere now.

**Pull request merged**